person.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+# Person class provides the default implementation of the correct_name method
+class Person < Nameable
+  def initialize(age, name = 'unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Getter and setter in one
```

Comment:
- [ ] nice job. please kindy ensure
Getters and Setters for @rentals should be present
 ```ruby
  attr_accessor :name, :age, :rentals
```